WalletWasabi/WabiSabi/Models/MultipartyTransaction/SigningState.cs

```diff
@@ -108,6 +126,10 @@ public Transaction CreateTransaction()
 		{
 			tx.Inputs[index].WitScript = witness;
 		}
+		foreach (var (index, witness) in UnpublishedWitnesses)
+		{
+			tx.Inputs[index].WitScript = witness;
+		}
```

Comment:
This is correct but I would suggest to have only one loop instead of having two anyway.

```suggestion
		foreach (var (index, witness) in Witnesses.Concat(UnpublishedWitnesses))
		{
			tx.Inputs[index].WitScript = witness;
		}

Reply:
ok, I did not like the two loops too.